docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,118 @@
+|{ecs-ref}/ecs-orchestrator.html[`orchestrator.*`] | `orchestrator.*`| ECS `orchestrator.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-organization.html[`organization.*`]| `organization.*`| ECS `organization.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-package.html[`package.*`]|`package.*`|ECS `package.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-process.html[`process.*`]| `process.*`| ECS `process.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-registry.html[`registry.*`] | `registry.*`| ECS `registry.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-related.html[`related.*`] |`related.*` |ECS `related.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-rule.html[`rule.*`]| `rule.*`| ECS `rule.*` fields copied from the source document when available.
+
+NOTE: These fields are not related to the detection rule that generated the alert.
+
+|{ecs-ref}/ecs-server.html[`server.*`] |`server.*`|ECS `server.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-service.html[`service.*`]|`service.*`|ECS `service.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-source.html[`source.*`] | `source.*` | ECS `source.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-tracing.html#field-span-id[`span.*`] |`span.*`|ECS `span.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-threat.html[`threat.*`]| `threat.*`|ECS `threat.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-tls.html[`tls.*`] | `tls.*`| ECS `tls.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-tracing.html[`trace.*`]| `trace.*`|ECS `trace.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-tracing.html#field-transaction-id[`transaction.*`]|`transaction.*`| ECS `transaction.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-url.html[`url.*`]|`url.*`|ECS `url.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-user.html[`user.*`]|`user.*`| ECS `user.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-user_agent.html[`user_agent.*`]|`user_agent.*`| ECS `user_agent.*` fields copied from the source document when available.
+|{ecs-ref}/ecs-vulnerability.html[`vulnerability.*`]|`vulnerability.*`|ECS `vulnerability.*` fields copied from the source document when available.
+|`signal.ancestors.*`|`kibana.alert.ancestors.*`|
+|`signal.depth`| `kibana.alert.depth`|
+|`signal.original_event.*` | `kibana.alert.original_event.*`|
+|`signal.original_time`|`kibana.alert.original_time`| The value copied from the source event (`@timestamp`).
+|`signal.reason` |`kibana.alert.reason`|
+|`signal.rule.author` |`kibana.alert.rule.author`|
```

Comment:
For all rows that start with `signal.rule.x` can we add a description like:
value of x from the rule that created this alert - see https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-ui-advanced-params or https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-ui-basic-params, depending on which matches?

Reply:
@MikePaquette I'm not quite sure I understand 100%. If I put a description of `signal.rule.building_block_type` according to this ^, wouldn't it read something like "The value of `building_block_type` from the rule that created this alert." Is that format correct for all the `signal.rule.x` fields?

Reply:
Good question. there are indeed a few fields, including `building_block_type` that are different from most of the others that start with `signal.rule.x`.  I see building_block_type, to, from, timeline_id, timeline_title, max_signals, that should be treated differently.  Let's treat them as special cases, and add a full definition of the field here, rather than pointing to the basic and advance settings.